### PR TITLE
Style Guide and Diataxis Review: Wallet V

### PR DESCRIPTION
**Pull request by charlesthompson3**
# Description of change

TW Style Guide and diataxis review has been applied to the Wallet documentation. The section reviewed was Python: Examples

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Documentation Fix

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by charlesthompson3** [comment]
I didn't want to tag on any other sections since this one is pretty substantial. I do want to note that the content/explanations are nearly 1:1 with the "Node.js: Examples" section; we could approach collapsing the example sections together with respective headers, some other idea, or leave it as is. Group consensus, basically @iotaledger/tech-writers

File: documentation/docs/libraries/python/examples.mdx [hunk 1]
```diff
@@ -21,63 +22,67 @@ import restore from  '!!raw-loader!./../../../../bindings/python/examples/6_rest
 import b_event_simple_event from  '!!raw-loader!./../../../../bindings/python/examples/7b_event_simple_event.py';
 import event_queue from  '!!raw-loader!./../../../../bindings/python/examples/7_event_queue.py';
 
-This section will guide you through several examples using the python binding of the `wallet.rs` library. You can also find the code for the examples in the `/bindings/python/examples` folder in the [official GitHub repository](https://github.com/iotaledger/wallet.rs/tree/dev/bindings/python/examples).
+In this section, you will go over several examples that use the python binding of the `wallet.rs` library. You can also find examples of the code in the `/bindings/python/examples` folder of the [official GitHub repository](https://github.com/iotaledger/wallet.rs/tree/dev/bindings/python/examples).
```
Comment (lucas-tortora) [inline documentation/docs/libraries/python/examples.mdx]:
```suggestion
In this section, you can find several examples that use the python binding of the `wallet.rs` library. You can also find examples of the code in the `/bindings/python/examples` folder of the [official GitHub repository](https://github.com/iotaledger/wallet.rs/tree/dev/bindings/python/examples).
```

File: documentation/docs/libraries/python/examples.mdx [hunk 2]
```diff
@@ -21,63 +22,67 @@ import restore from  '!!raw-loader!./../../../../bindings/python/examples/6_rest
...
 
-All the examples in this section expect you to set your custom password  in the _.env_ file:
+All of the examples in this section are based on you having set your custom password in the _.env_ file:
```
Comment (lucas-tortora) [inline documentation/docs/libraries/python/examples.mdx]: I'm not sure this change works better than the original

**Review by lucas-tortora — APPROVED** [review]

**Comment by lucas-tortora** [comment]
> I didn't want to tag on any other sections since this one is pretty substantial. I do want to note that the content/explanations are nearly 1:1 with the "Node.js: Examples" section; we could approach collapsing the example sections together with respective headers, some other idea, or leave it as is. Group consensus, basically @iotaledger/tech-writers

We have a lot of overlapping  text with the node examples here, and both have a mix of explanations and how-tos.  Personally, I'd separate explanations into a key concepts section, group the examples by action rather than language, add "tabs" for the users to choose the language and reference links to the explanations in the how tos and vice-versa.   Wdyt?

**Comment by charlesthompson3** [comment]
> > I didn't want to tag on any other sections since this one is pretty substantial. I do want to note that the content/explanations are nearly 1:1 with the "Node.js: Examples" section; we could approach collapsing the example sections together with respective headers, some other idea, or leave it as is. Group consensus, basically @iotaledger/tech-writers
> 
> We have a lot of overlapping text with the node examples here, and both have a mix of explanations and how-tos. Personally, I'd separate explanations into a key concepts section, group the examples by action rather than language, add "tabs" for the users to choose the language and reference links to the explanations in the how tos and vice-versa. Wdyt?

Initial reaction to the group examples by action rather than language made me wince but I think I need to quick draft it to really see how it would look. Otherwise, I don't have a problem with this approach. As long as its efficient and helpful for the readers, I'm game.

@aleksei-korolev might have some more insight, so I'll leave this one open for a bit; yes?

**Comment by aleksei-korolev** [comment]
At the very least we should separate explanations, how-tos and references. But examples themselves are not a type of an article, they can be part of anything. 

Regarding action-first vs language-first, I see no big difference in UX between clicking on "python" and then "check balance" over clicking "check balance" and then "python", so I would go with the one that is easier to maintain (probably action-first, since it will probably have less text duplication this way).

**Comment by thibault-martinez** [comment]
Hello @iotaledger/tech-writers, what's the status of this PR ?

**Comment by charlesthompson3** [comment]
@thibault-martinez all clear 👍

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**